lib/ansible/module_utils/facts.py

```diff
@@ -1308,6 +1308,17 @@ def get_device_facts(self):
                     if not part['sectorsize']:
                         part['sectorsize'] = get_file_content(part_sysdir + "/queue/hw_sector_size",512)
                     part['size'] = self.module.pretty_bytes((float(part['sectors']) * float(part['sectorsize'])))
+                    part['holders'] = []
```

Comment:
since this is basically a copy of the existing code for devices, i would make it into a function and call it from both spots